pkgs/by-name/vi/vikunja-desktop/default.nix

```diff
@@ -0,0 +1,136 @@
+{ lib
+, stdenv
+, makeWrapper, makeDesktopItem, callPackage
+, pnpm, nodejs
+, electron
+, unstableGitUpdater
+, fetchFromGitHub
+, fetchzip
+}:
+
+let
+  executableName = "vikunja-desktop";
+  version = "0.24.3";
+  src = fetchFromGitHub {
+    owner = "go-vikunja";
+    repo = "vikunja";
+    rev = "v${version}";
+    hash = "sha256-UT2afhjEangilvflmxxahj7pEiJUWxqUL1Eni1JvuRI=";
+  };
+
+  frontend = stdenv.mkDerivation (finalAttrs: {
```

Comment:
This is duplicated with the vikunja main package. I wonder if we could resonably abstract this?

Reply:
Either make it a top-level attribute, or put it into the passthru attrset of another package and consume it as other_package.frontend.

Reply:
Done!